packages/blocks/src/components/SplitButton/SplitButton.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { AutoImportIcon, CaretDownIcon } from '@mergestat/icons';
+
+type SplitButtonProps = {
+  children?: React.ReactNode;
+  onClickClose?: () => void;
+  onClick?: () => void;
+};
+
+export const SplitButton: React.FC<SplitButtonProps> = ({
+  children,
+  onClickClose,
+  onClick,
+}) => {
+  return (
+    <div className="t-flex-split t-split-gray text-base">
+      <label onClick={onClick}>
+        <AutoImportIcon className="t-icon" />
+        <button>{children || 'Create Auto Import'}</button>
+      </label>
+      <div className="t-divider" />
+      <CaretDownIcon className="t-icon" onClick={onClickClose} />
+    </div>
+  );
+};
```

Comment:
The two sections of the split button should be independent, like two separate buttons that are just next to each other and logically grouped. When I click now, it looks like the whole thing is pressed/clicked (both buttons) when it should be one segment or the other. Does that make sense? The actions of the first part and second part will usually be different - the second part will usually open up a menu

<img width="244" alt="image" src="https://user-images.githubusercontent.com/57259/163597211-009b9d56-0475-4829-a660-1b327e2976be.png">


<img width="244" alt="image" src="https://user-images.githubusercontent.com/57259/163597152-e0528aa4-b09f-458b-874f-dbe28f953d44.png">